src/SwedbankPay.Episerver.Checkout.OrderManagement/Steps/CancelPaymentStep.cs

```diff
@@ -103,10 +103,26 @@ public override async Task<PaymentStepResult> Process(IPayment payment, IOrderFo
 
             if (Successor != null)
             {
-                return await Successor.Process(payment, orderForm, orderGroup, shipment).ConfigureAwait(false);
+                return Successor.Process(payment, orderForm, orderGroup, shipment);
             }
 
             return paymentStepResult;
         }
+
+
+        private void A(IOrderGroup orderGroup, string noteTitle, string noteMessage)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'A'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5473830)